src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -41,17 +42,58 @@ public void init() {
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Map<String, String> roomProperties = readJSONFromRequest(request);
 
-    // Save player roomId to datastore.
+    // Save roomId to datastore.
     Entity roomEntity = new Entity("Room");
     roomEntity.setProperty("roomId", roomProperties.get("roomId"));
+    roomEntity.setProperty("userId", roomProperties.get("userId"));
```

Comment:
Is there any way to store multiple userId's? You might want to look into storing a list of userId's (you can store ArrayList in Datastore)